crypto/kzg/src/lib.rs

```diff
@@ -66,7 +66,9 @@ impl Kzg {
         Ok(Self {
             trusted_setup: KzgSettings::load_trusted_setup(
                 &trusted_setup.g1_points(),
+                &trusted_setup.g1_monomial_points(),
                 &trusted_setup.g2_points(),
+                0,
```

Comment:
Here I'm using 0 as precomputation value; this is a new parameter in v2 so I'm not sure what it should be...